Bottom: 460ac712802228518e3dc95117e949f741245168
Top:    524a8a31657b2cd98a8e4d0a54ca0187f6ce86f6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-09 22:50:25 +0000

Refresh of try-a-new-different-sized-vbo-

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 02199da..e5cbc4a 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1045,8 +1045,6 @@ ghid_end_drawing (GHidPort *port)
   else
     glFlush ();
 
-  gdk_gl_drawable_wait_gl (pGlDrawable);
-
   /* end drawing to current GL-context */
   gdk_gl_drawable_gl_end (pGlDrawable);
 }
